### PR TITLE
"This doesn't belong here?" Brings bay's spare id to our maps.

### PR DESCRIPTION
**Pull request by praisenarsie**
Since Bay is removing their spare id 
I might as well add it to our maps...

million dollar joke

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Just use our gold ID instead of mixing and match old-style sprites with new ones. Then clown can play pranks too.

**Comment by praisenarsie** [comment]
also pubby's mapdiff is long because there was an area placed in the wrong spot

**Comment by DaxDupont** [comment]
Add it to loot table instead

**Comment by PKPenguin321** [comment]
anybody else low key get hit with nostalgia by that sprite

**Comment by praisenarsie** [comment]
@PKPenguin321 one of the reasons why i got the sprite from bay instead of using our shinier one

**Comment by MMMiracles** [comment]
for something that'll be on every map should be its own thing instead of just a variety edited id, tbh

**Comment by lzimann** [comment]
what mmmiracles said

**Review by lzimann — CHANGES_REQUESTED** [review]
make it an actual path instead of varedit across 5 maps

**Comment by ExcessiveUseOfCobblestone** [comment]
I've literally never noticed the spare is not this sprite anymore.

**Review by lzimann — APPROVED** [review]

**Comment by ShizCalev** [comment]
conflict.

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -360,3 +360,11 @@ update_label("John Doe", "Clowny")
...
 	icon_state = "centcom"
 	access = list(ACCESS_ENGINE_EQUIP)
+
+/obj/item/card/id/bay
+		name = "commanding officer's spare id"
+		desc = "This feels as if it doesn't belong here...?"
+		icon_state = "classicgold"
+		item_state = "classicgold_id"
+		lefthand_file = 'icons/mob/inhands/equipment/idcards_lefthand.dmi'
+		righthand_file = 'icons/mob/inhands/equipment/idcards_righthand.dmi'
```
Comment (ACCount12) [inline code/game/objects/items/cards_ids.dm]: Does it even need those? Aren't they set in parent type already?

**Comment by praisenarsie** [comment]
@DaxDupont told me i have to rebase due to coiax doing funky shit with mapping

tldr have to do a lot of git fuckery, will just remake it in a new pr because i'm not bothered to rebase and fix conflicts and all
